src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,47 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  title: string,
+  description: string,
+  imgUrl: string,
+  imdbUrl: string,
```

Comment:
beter pass `movie` as one prop and use your `Movie` type here so you don't repeat yourself